app/views/tools/show.html.erb

```diff
@@ -0,0 +1,29 @@
+<% provide(:title, t('tool.title')) %>
+
+<div class="col-md-8 col-md-offset-2 col-sm-12">
+  <div class="headline">
+    <h1><%= @tool.title %></h1>
+  </div>
+  <p><%= @tool.description %></p>
+  <p><b><%= t('tool.total_free') %>: </b><%= @tool.total %></p>
+  <p><b><%= t('tool.free') %>: </b><%= @tool.free %></p>
+
+  <div class="headline">
+    <h3><%= t('tool.borrowed') %></h3>
+  </div>
+
+  <ul class="list-group">
+    <% @tool.tool_rentings.each do |rent|
+      if rent.returned
+        next
```

Comment:
I tried that but it seemed that @tool.tool_rentings took everything anyways. But I will try it again